stratum/hal/bin/bcm/standalone/BUILD

```diff
@@ -102,3 +103,49 @@ pkg_tar(
     ],
     mode = "0755",
 )
+
+pkg_tar(
+    name = "stratum_bcm_bin",
+    srcs = [
+        ":stratum_bcm",
+        "start-stratum.sh",
+        "load-kernel-modules.sh",
+    ],
+    mode = "0755",
+    package_dir = "/usr/bin",
```

Comment:
TODO: Coordinate on the paths we use @Yi-Tseng

Reply:
As far as I know the `/usr/...` is mostly for system built-in stuff. While the `/usr/local/...`  for a system admin to install the software package locally.
We can use `/usr` if we think the Stratum is a system built-in software package. And use `/usr/local` if it is not.
I think we can use `/usr` since the Stratum will be a built-in package in the feature. (e.g., add PPA to our ONL image and install it when we are building it)

Reply:
Looking at the Debian [package guide](https://www.debian.org/doc/debian-policy/ch-opersys#site-specific-programs), it seems `/usr/local/` is prohibited in packages:

> As mandated by the FHS, packages must not place any files in /usr/local, either by putting them in the file system archive to be unpacked by dpkg or by manipulating them in their maintainer scripts.